script.js

```diff
@@ -0,0 +1,446 @@
+		} else {
+			keep = false;
+			alert(
+				`No, the person doesn't wear ${currentQuestion.value}! Remove all people that wears ${currentQuestion.value}`
+			);
+		}
+	}
+	if (currentQuestion.category.includes("other")) {
+		if (secret[currentQuestion.category].includes(currentQuestion.value)) {
+			keep = true;
+
+			alert(
+				`Yes, the person is a ${currentQuestion.value}! Keep all people that are ${currentQuestion.value}s`
+			);
+		} else {
+			keep = false;
+			alert(
+				`No, the person is not a ${currentQuestion.value}! Remove all people that are ${currentQuestion.value}s`
+			);
+		}
+	}
+
+	if (currentQuestion.category.includes("hair")) {
+		if (secret[currentQuestion.category].includes(currentQuestion.value)) {
+			keep = true;
+
+			alert(
+				`Yes, the person has ${currentQuestion.value} hair! Keep all people with ${currentQuestion.value} hair`
+			);
+		} else {
+			keep = false;
+			alert(
+				`No, the person doesn't have ${currentQuestion.value} hair! Remove all people that have ${currentQuestion.value} hair`
+			);
+		}
+	}
+	if (currentQuestion.category.includes("eyes")) {
+		if (secret[currentQuestion.category].includes(currentQuestion.value)) {
+			keep = true;
+			alert(
+				`Yes, the person has ${currentQuestion.value} eyes! Keep all people with ${currentQuestion.value} eyes`
+			);
+		} else {
+			keep = false;
+			alert(
+				`No, the person doesn't have ${currentQuestion.value} eyes! Remove all people that have ${currentQuestion.value} eyes`
+			);
+		}
+	}
+
+	if (currentQuestion.category.includes("age")) {
+		if (secret[currentQuestion.category].includes(currentQuestion.value)) {
+			keep = true;
+			alert(
+				`Yes, the person are ${currentQuestion.value}! Keep all the ${currentQuestion.value} people`
+			);
+		} else {
+			keep = false;
+			alert(
+				`No, the person is not ${currentQuestion.value}! Remove all the ${currentQuestion.value} people`
+			);
+		}
+	}
+
+	if (currentQuestion.value !== "") {
+		if (keep) {
+			charactersInPlay = charactersInPlay.filter((person) =>
+				person[currentQuestion.category].includes(currentQuestion.value)
+			);
+		} else {
+			charactersInPlay = charactersInPlay.filter(
+				(person) =>
+					!person[currentQuestion.category].includes(currentQuestion.value)
+			);
+		}
+	} else {
+		if (keep) {
+			charactersInPlay = charactersInPlay.filter(
+				(person) => person[currentQuestion.category] === value
+			);
+		} else {
+			charactersInPlay = charactersInPlay.filter(
+				(person) => person[currentQuestion.category] !== value
+			);
+		}
+	}
+
+	generateBoard();
+};
+
+const guess = (suspected) => {
+	const yourGuess = prompt(
+		`So you think ${suspected} is the suspected person? (yes or no) `
+	);
+
+	if (yourGuess === "yes") {
+		board.style.display = "none";
+		winOrLose.style.display = "flex";
+		if (secret.name === suspected) {
+			alert(`you win`);
+		} else if (secret.name !== yourGuess) {
+			alert(`you lose! This time the "Guess-Who-person" was ${secret.name}`);
```

Comment:
I was wondering if you can put the value of the name asking in the beginning of the game you can use here with ${ }

Reply:
Hi, I tried that but dint make it work.. do you have any solution to make that work? :)

to inser the ${addName} not working  


Tacks for your feedback!